### DISCUSSION TITLE
Welcome to code-1 Discussions!

### DISCUSSION THREAD

**Discussion by corgiboss9gamer** [corgiboss9gamer/code-1#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by corgiboss9gamer** [comment]
what should we do with this code?

**Reply by kmikuska** [reply]
I think we should start looking at raspberry pi code.

**Reply by corgiboss9gamer** [reply]
okay how??
